.github/workflows/ci.yml

```diff
@@ -359,6 +359,7 @@ jobs:
             !:trino-jdbc,
             !:trino-kafka,
             !:trino-kudu,
+            !:trino-lancedb,
```

Comment:
should this be called `trino-lancedb`, or more like `trino-lance` since it's really about reading and writing the Lance format through Trino?